src/qt/walletmodel.cpp

```diff
@@ -65,7 +65,8 @@ WalletModel::~WalletModel()
 void WalletModel::startPollBalance()
 {
     // This timer will be fired repeatedly to update the balance
-    connect(timer, &QTimer::timeout, this, &WalletModel::pollBalanceChanged);
+    connect(timer, &QTimer::timeout, this, &WalletModel::timerTimeout);
```

Comment:
Might want to add a comment here why the signal is not directly connected (e.g. because `QTimer::timeout` is a private signal)

Reply:
Thanks! [Updated](https://github.com/bitcoin-core/gui/pull/260#issuecomment-813403971).